gpytorch/priors/torch_priors.py

```diff
@@ -30,6 +30,28 @@ def expand(self, batch_shape):
         return NormalPrior(self.loc.expand(batch_shape), self.scale.expand(batch_shape))
 
 
+class HalfNormalPrior(Prior, HalfNormal):
+    """
+    Half-Normal prior.
+    
+    pdf(x) = 2 * (2 * pi * sigma^2)^-0.5 * exp(-x^2 / (2 * sigma^2)) for x >= 0; 0 for x < 0
+
+    where sigma^2 is the variance.
```

Comment:
nit: can we use `scale` or `sigma` consistently between documentation and method arguments?